authors/apps/authentication/tests/test_user_registration.py

```diff
@@ -29,58 +29,92 @@ def setUp(self):
             "user": {
                 "username": "",
                 "email": "jake@jake.jake",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.signup_empty_email = {
             "user": {
-                "username": "gigz",
+                "username": "Marygigz",
                 "email": "",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.signup_empty_password = {
             "user": {
-                "username": "gigz",
+                "username": "Marygigz",
                 "email": "jake@andela.com",
                 "password": ""
                 }}
         self.signup1 = {
             "user": {
-                "username": "gigz",
-                "email": "jake@andela.com",
-                "password": "jakejake"
+                "username": "Marygigz",
+                "email": "jake@jake.jake",
+                "password": "g@_Gigz-2416"
                 }}
         self.signup2 = {
             "user": {
-                "username": "jake",
+                "username": "Maryjake",
                 "email": "jake@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.signup3 = {
             "user": {
-                "username": "jake",
+                "username": "Maryjake",
                 "email": "jakey@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.duplicate_email = {
             "user": {
-                "username": "jake",
+                "username": "Gigzjake",
                 "email": "jacky@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.duplicate_email2 = {
             "user": {
-                "username": "jacky2",
+                "username": "jacky2gigz",
                 "email": "jacky@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
-
+        self.username_characters_only = {
+                                            "user": {
+                                                "username": "#@##@#@##@#",
+                                                "email": "jake@jake.jake",
+                                                "password": "g@AHJA473"
+                                                }}
```

Comment:
The indentation of the user objects seems off... Can you just give the objects just four spaces? like Maggie's

Reply:
I am working on this too